repos/spack_repo/builtin/packages/py_astroid/package.py

```diff
@@ -68,5 +70,6 @@ class PyAstroid(PythonPackage):
     depends_on("py-typing-extensions@4.0:", when="@2.14: ^python@:3.10", type=("build", "run"))
     depends_on("py-setuptools@17.1:", type=("build", "run"))
     depends_on("py-setuptools@20:", when="@2.7.3:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-setuptools@17.1:", type="build")
    depends_on("py-setuptools@20:", when="@2.7.3:", type="build")
```
I think the setuptools dependency is only needed for installation